chore: changelog 49.9.0 (#2503)
forcedotcom/salesforcedx-vscode: packages/salesforcedx-vscode/CHANGELOG.md

# 49.9.0 - September 10, 2020

> ## Added

Reviewer: ```suggestion

## Added
```

# 49.9.0 - September 10, 2020

> 

Reviewer: ```suggestion
## Added

### salesforcedx-vscode-apex
- Added support for the new [Safe Navigator Operator](https://releasenotes.docs.salesforce.com/en-us/winter21/release-notes/rn_apex_SafeNavigationOperator.htm) within the Apex Language Server Protocol ([PR #2507](https://github.com/forcedotcom/salesforcedx-vscode/pull/2507))

```

## Fixed

#### salesforcedx-vscode-lightning

> - chore: update to latest lwc/aura lang server version ([PR #2495](https://github.com/forcedotcom/salesforcedx-vscode/pull/2495))

Reviewer: I would remove this section

> #### salesforcedx-vscode-lwc

Reviewer: ```suggestion
#### salesforcedx-vscode-lightning, salesforcedx-vscode-lwc
```

> - chore: update to latest lwc/aura lang server version ([PR #2495](https://github.com/forcedotcom/salesforcedx-vscode/pull/2495))

Reviewer: ```suggestion
- Update to the latest version of the Aura and LWC language servers ([PR #2495](https://github.com/forcedotcom/salesforcedx-vscode/pull/2495))
```

#### salesforcedx-vscode

- chore: changelog 49.8.0 updates ([PR #2486](https://github.com/forcedotcom/salesforcedx-vscode/pull/2486))

#### docs

> - Updates to Nate's [code completion](https://developer.salesforce.com/tools/vscode/en/apex/writing) documentation updates (minor edits) ([PR #2471](https://github.com/forcedotcom/salesforcedx-vscode/pull/2471))

Reviewer 2: ```suggestion
- Updates to [Code Completion](https://developer.salesforce.com/tools/vscode/en/apex/writing) documentation ([PR #2471](https://github.com/forcedotcom/salesforcedx-vscode/pull/2471))
```

> - Added [link to community extensions](https://developer.salesforce.com/tools/vscode/en/getting-started/recommended-extensions) ([PR #2477](https://github.com/forcedotcom/salesforcedx-vscode/pull/2477))

Reviewer 2: ```suggestion
- Added [Community Extensions](https://developer.salesforce.com/tools/vscode/en/getting-started/recommended-extensions/#community-extensions) section ([PR #2477](https://github.com/forcedotcom/salesforcedx-vscode/pull/2477))
```
